pkgs/development/tools/build-managers/tup/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, fuse, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "tup-v${version}";
```

Comment:
You should leave off the 'v' prefix, as it prevents Nix from being able to parse the version correctly:

    $ nix-repl
    Welcome to Nix version 1.8. Type :? for help.
    
    nix-repl> builtins.parseDrvName "foo-v1.2.3"
    { name = "foo-v1.2.3"; version = ""; }

    nix-repl> builtins.parseDrvName "foo-1.2.3"
    { name = "foo"; version = "1.2.3"; }

I'll make that change and merge myself.